.coveragerc

```diff
@@ -10,6 +10,9 @@ show_missing = True
 precision = 2
 omit =
         *__init__*
+        */tests/*
+        *tests/*
+        */tests*
         *tests*
```

Comment:
I don't know which of these lines worked to fix the bug. I propose we keep these.